### ISSUE TITLE
Crash when using deleted tags

### ISSUE THREAD

**Issue by mortii** [mortii/anki-morphs#161]
> In the process of troubleshooting this issue, I encountered another issue. I created a new tag and then specified it in the "include" for the note filters so that I could narrow down which cards were causing it to crash. Once I figured it out, I deleted that new tag without removing it from the note filter first, which caused it to crash. I had to go back and re-create that tag, remove it from the note filter, and then I could delete that tag.

_Originally posted by @xofm31 in https://github.com/mortii/anki-morphs/issues/159#issuecomment-1962927516_

**Comment by mortii** [comment]
Ah, it crashes when clicking the tags cell in the note filter settings, and gives this error:

```
Traceback (most recent call last):
  File "/home/neo/.local/share/Anki2/addons21/472573498/settings_dialog.py", line 796, in _tags_cell_clicked
    self.tag_selector.set_selected_tags_and_row(
  File "/home/neo/.local/share/Anki2/addons21/472573498/tag_selection_dialog.py", line 88, in set_selected_tags_and_row
    _include_checkbox: QCheckBox = get_checkbox_widget(
  File "/home/neo/.local/share/Anki2/addons21/472573498/table_utils.py", line 19, in get_checkbox_widget
    assert isinstance(widget, QCheckBox)
AssertionError
```

Just catching the exception and ignoring it should be fine I think. I'll try fixing it.

**Comment by mortii** [comment]
@xofm31 Fixed in [v1.2.3](https://github.com/mortii/anki-morphs/releases/tag/v1.2.3), thanks!
